### ISSUE TITLE
Pretty print package objects

### ISSUE THREAD

**Issue by khusmann** [frictionlessdata/frictionless-r#154]
Thanks for all your work on this package! I really appreciate the simplicity & tidyverse principles. I hope you don't mind if I jump in!

Presently, printing loaded packages in the console simply dumps their list representation (which can span many pages in larger packages). I think it would be really nice for packages to have a pretty-printed summary by default instead. This is handy for interactive sessions, gives the package a really polished look, and I think it would help with adoption efforts by making things a little less intimidating for new users. 

I'll submit a pull request in a sec with a candidate implementation...

**Comment by peterdesmet** [comment]
I agree! Makes me also wonder if we should move the console message one currently gets when using `read_package()` to the print function. The message is a bit annoying for packages relying on this package. Ping @damianooldoni.

**Comment by khusmann** [comment]
> Makes me also wonder if we should move the console message one currently gets when using read_package() to the print function.

Oh, I like this idea! I definitely found the default loading message a little annoying for exactly that reason, and was planning to request an option to disable it. It feels a lot more natural in a `print` function.

**Comment by peterdesmet** [comment]
Question: how would a user get the default (non-pretty) print? This might especially be useful for developers building on top of frictionless. And what would the result be of:

```R
str(package, list.len = 3)
```

**Comment by khusmann** [comment]
> how would a user get the default (non-pretty) print?

Ah, good thought. As a user I would expect to do something like `as.list(example_package)` to remove the datapackage class. If you like that approach lmk and I can add.

> `> str(pkg, list.len=3)`

```
List of 8
 $ name     : chr "example_package"
 $ id       : chr "115f49c1-8603-463e-a908-68de98327266"
 $ created  : chr "2021-03-02T17:22:33Z"
  [list output truncated]
 - attr(*, "class")= chr "datapackage"
```

**Comment by peterdesmet** [comment]
`as.list()` sounds like a good approach. @damianooldoni would you agree?

**Comment by peterdesmet** [comment]
This is now implemented in #184. The `message()` is removed from `read_package()`. To get the default print, `unclass()` is suggested, not `as.list()`.
